<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>561</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>441</width>
    <height>561</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>账号信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
    background-color: #fff; 
    border: none; 
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_16">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>281</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
font: 16pt &quot;华文行楷&quot;;
   background-color:rgba(0,0,0,0);
   border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_17">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>281</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
font: 16pt &quot;华文行楷&quot;;
   background-color:rgba(0,0,0,0);
   border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文行楷&quot;;
background-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文行楷&quot;;
background-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>手机号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>161</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;华文行楷&quot;;
background-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>密码更改：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_18">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>281</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
font: 16pt &quot;华文行楷&quot;;
   background-color:rgba(0,0,0,0);
   border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>161</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文行楷&quot;;
background-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>原密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>161</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文行楷&quot;;
background-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_19">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>281</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
font: 16pt &quot;华文行楷&quot;;
   background-color:rgba(0,0,0,0);
   border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/Lenovo/.designer/res_img/古风01.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文行楷&quot;;
background-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_update_password">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>181</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
   </property>
   <property name="text">
    <string>更改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 0);
    color: black;
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 0);
    color: black;
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255,80);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Lenovo/.designer/res_img/退出.png</normaloff>C:/Users/Lenovo/.designer/res_img/退出.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;华文行楷&quot;;
background-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>lineEdit_16</zorder>
  <zorder>lineEdit_17</zorder>
  <zorder>label_20</zorder>
  <zorder>label_21</zorder>
  <zorder>label_22</zorder>
  <zorder>lineEdit_18</zorder>
  <zorder>label_23</zorder>
  <zorder>label_24</zorder>
  <zorder>lineEdit_19</zorder>
  <zorder>label_25</zorder>
  <zorder>btn_update_password</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label_26</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
